charts/vela-core/templates/defwithtemplate/daemon.yaml

```diff
@@ -93,7 +93,7 @@ spec:
         	] | []
 
         	secret: *[
-        		for v in parameter.volumeMounts.secret {
+        		if parameter.volumeMounts != _|_ && parameter.volumeMounts.secret != _|_ for v in parameter.volumeMounts.secret {
```

Comment:
Will generate an error if without this additional judgement?

Reply:
Yes, refer to this issue:  cue-lang/cue#1815